### PR TITLE
Fix torque adc zero offsets

### LINKED ISSUE #135: Determine offsets for analog input channels
I just realized that we've never verified that what we think is zero is actually zero.

### PR DESCRIPTION
**Pull request by oliverlee**
Add constants defining the ADC value for a corresponding zero torque.
Fix Kistler torque sensor and Kollmorgen actual torque measurements.

Fixes #135

### CODE REVIEW

File: projects/flimnap/main.cc [hunk 1]
```diff
@@ -89,6 +89,14 @@ namespace {
         return aout;
     }
 
+    constexpr float convert_adc_to_Nm(adcsample_t value, adcsample_t adc_zero, float magnitude) {
+        // ADC samples are 12 bits
+        // It's not clear when scaling should be applied as data was never saved after the scale
+        // factors were determined.
+        const int16_t shifted_value = static_cast<int16_t>(value) - static_cast<int16_t>(adc_zero);
+        return static_cast<float>(shifted_value)*magnitude/2048;
```
Comment (mickvangelderen) [inline projects/flimnap/main.cc]: Use `2048.0f` in floating point calculations.

File: projects/flimnap/main.cc [hunk 2]
```diff
@@ -89,6 +89,14 @@ namespace {
         return aout;
     }
 
+    constexpr float convert_adc_to_Nm(adcsample_t value, adcsample_t adc_zero, float magnitude) {
```
Comment (mickvangelderen) [inline projects/flimnap/main.cc]: Drop `convert_`, thats clear from the word `to`. I'd consider making it all lower case for consistency. `adc_to_nm` and add a comment about what it does if necessary.

File: projects/flimnap/main.cc (hunk 1 above)
Comment (oliverlee) [inline projects/flimnap/main.cc]: fixed

File: projects/flimnap/main.cc (hunk 2 above)
Comment (oliverlee) [inline projects/flimnap/main.cc]: fixed

File: scripts/plot_sim.py [hunk 3]
```diff
@@ -8,8 +8,11 @@
 
 from load_sim import load_messages, get_records_from_messages, get_time_vector
 
+MAX_KISTLER_TORQUE = 50
```
Comment (mickvangelderen) [inline scripts/plot_sim.py]: Fine for now, creating an issue to deduplicate these values in some way.

Comment (mickvangelderen) [inline scripts/plot_sim.py]: #139

**Pull request closed without merging**